# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/HTMLEditor.h
# Commit: d6c4fdcf6a69
# Full Hash: d6c4fdcf6a69ebaa1b133ef8c57da9eef389f89a
# Author: Tom Schuster <evilpies@gmail.com>
# Date: 2022-10-20 21:51:26
# Regressor Bug: 1778565
# File Overlap Count: 1
# Description:
#   Bug 1778565 - Editor: Handle nsIFile transferable as blob. r=nika,masayuki
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D155753
# ==============================================================================

diff -r 88976c36da0a -r d6c4fdcf6a69 editor/libeditor/HTMLEditor.h
--- a/editor/libeditor/HTMLEditor.h	Thu Oct 20 06:59:31 2022 -0400
+++ b/editor/libeditor/HTMLEditor.h	Thu Oct 20 11:11:59 2022 +0000
@@ -4255,10 +4255,10 @@
    * operation is finished.
    *
    * @param aBlob       The input blob
-   * @param aWindow     The global object under which the read should happen.
+   * @param aGlobal     The global object under which the read should happen.
    * @param aBlobReader The blob reader object to be notified when finished.
    */
-  static nsresult SlurpBlob(dom::Blob* aBlob, nsPIDOMWindowOuter* aWindow,
+  static nsresult SlurpBlob(dom::Blob* aBlob, nsIGlobalObject* aGlobal,
                             BlobReader* aBlobReader);
 
   /**